frontend/public/module/k8s/resource.js

```diff
@@ -118,6 +118,9 @@ export const k8sKill = (kind, resource, opts = {}, json = null) =>
     json,
   );
 
+export const k8sKillByName = (kind, name, namespace, opts = {}) =>
+  k8sKill(kind, { metadata: { name, namespace } }, opts);
```

Comment:
nit: It would be better to have `k8sKill` call `k8sKillByName` instead of the other way around so we don't have to create a fake object. I realize this is how `k8sPatchByName` was written, though. We could clean this up in a follow on.